library/src/main/java/io/getstream/chat/android/client/models/EventType.kt

```diff
@@ -37,7 +44,6 @@ object EventType {
     val NOTIFICATION_MARK_READ = "notification.mark_read"
     val NOTIFICATION_INVITED = "notification.invited"
     val NOTIFICATION_INVITE_ACCEPTED = "notification.invite_accepted"
-    val NOTIFICATION_INVITE_REJECTED = "notification.invite_rejected"
```

Comment:
why are we dropping this one?

Reply:
This event doesn't exist in backend. It seems it is an old one that is not used anymore